Remove progress blockers from iOS docs. (#766)
spatialos/gdk-for-unity: docs/content/mobile/ios/cloud-deploy.md

You know it’s complete when it says `Completed build for Cloud target` in your Unity Editor’s Console window.
1. Select **SpatialOS** > **Launch mobile client** > **iOS Device**.
1. In Finder, navigate to `/workers/unity/build/worker/` and locate the `.xcodeproj` that corresponds to your iOS client-worker, it may be in a sub-folder.<br>
	Open it in Xcode.
1. Still in XCode, select the **Play** button in the top left of the window.
1. Once the game is deployed and started on the Simulator, you see an empty text field and a **Connect** button: Select **Connect**.<br>
Note: You don’t need to enter anything in the text field.
1. Play the game on the Simulator.
1. When you're done, exit the Simulator.<br>
	In XCode, select the **Play** button to stop your the game running.<br>
	In your SpatialOS Console, select **Stop** to halt your cloud deployment.

## iOS device{#ios-device}

1. Make sure your computer and your mobile device are both connected to the same network.
1. Connect the mobile device to your computer using a USB cable.
1. Open your project in your Unity Editor.

> 1. In your Unity Editor, select **File** > **Build Settings**, and ensure that **iOS** is selected. Selection is indicated by a Unity logo that appears next to the name of the selected platform.<br>

Reviewer: ```suggestion
1. In your Unity Editor, select **File** > **Build Settings**, and ensure that **iOS** is selected. (The Unity logo next to a platform name indicates it's selected.)<br>
```
Slight rejig suggestion to make the text more friendly and active voice. Not really a biggie.
